resin/resin/imgui/sdf_tree.cpp

```diff
@@ -166,21 +169,27 @@ void SDFTreeComponentVisitor::visit_group(::resin::GroupNode& node) {
         if (ImGui::MenuItem("OBJ")) {
           ::resin::FileDialog::instance().save_file(
               [curr_id, &sdf_tree, resolution](const std::filesystem::path& path) {
-                ::resin::MeshExporter exporter(resolution);
+                ::resin::ShaderResourceManager resource_manager = ::resin::ResourceManagers::shader_manager();
+                ::resin::ShaderResource shader_resource =
+                    *resource_manager.get_res(std::filesystem::current_path() / "assets/marching_cubes.comp");
+                ::resin::MeshExporter exporter(shader_resource, resolution);
                 glm::vec3 pos = sdf_tree.group(curr_id).transform().pos();  // TODO(SDF-130) calculate bounding box
-                exporter.setup_scene(pos - glm::vec3(5.0f), pos + glm::vec3(5.0f), sdf_tree, curr_id);
-                exporter.export_mesh(path.string(), "obj");
+                exporter.setup_scene(pos - glm::vec3(5.0F), pos + glm::vec3(5.0F), sdf_tree, curr_id);
+                exporter.export_mesh(path, "obj");
               },
               std::span<const ::resin::FileDialog::FilterItem>(kMeshFiltersArray), std::string(name) + ".obj");
         }
 
         if (ImGui::MenuItem("GLTF")) {
           ::resin::FileDialog::instance().save_file(
               [curr_id, &sdf_tree, resolution](const std::filesystem::path& path) {
-                ::resin::MeshExporter exporter(resolution);
+                ::resin::ShaderResourceManager resource_manager = ::resin::ResourceManagers::shader_manager();
```

Comment:
Avoid copying the resource manager:
```suggestion
auto& resource_manager = ::resin::ResourceManagers::shader_manager();
```

Reply:
done